config/routes.rb

```diff
@@ -159,9 +159,9 @@
     resources :employer_applications do
       put :terminate
       put :cancel
-      put :reinstate
       collection do
-        get :get_term_reasons
+        get :term_reasons
```

Comment:
why this change from `get_term_reasons` to `term_reasons` ? 

did `rubocop` complained about it when you tried to commit ?

Reply:
yes, rubocop was complaining about it and I have cross-verified with DC enroll and it was defined with term_reasons method name.

Reply:
then, we might have a bigger problem here - if rubocop is complaining about changes that you didn't modified.

Reply:
No, this action `get_term_reasons` is defined in app/controllers/exchanges/employer_applications_controller.rb, as I have changed it on controller action due to rubocop issue I had to change it wherever this action/method is been used.